api/src/services/setup/upgrade.js

```diff
@@ -3,11 +3,14 @@ const upgradeLog = require('./upgrade-log');
 const upgradeSteps = require('./upgrade-steps');
 const logger = require('../../logger');
 
+let upgrading;
```

Comment:
Well that's the thing. 
I was this to reset if the server is restarted. The upgrade log, as it works now, will always get it from the DB, which means that we do have a valid upgrade log for an interrupted upgrade.
Do you have another suggestion?

Reply:
Why do you want it to clear on restart?

Reply:
Because whatever processes that were happening have clearly stopped. 
We could have interrupted before views were staged for example. 
Resuming is a bigger piece of work.

Reply:
I edited the above message a couple of times. sorry.

Reply:
... discussion continued in Slack ...